grpc/src/test/java/com/linecorp/armeria/server/grpc/GrpcServiceTest.java

```diff
@@ -119,4 +124,16 @@ public void missingMethod() throws Exception {
                            .set(HttpHeaderNames.CONTENT_LENGTH, "0"),
                 HttpData.of(new byte[] {})));
     }
+
+    @Test
+    public void pathMapping() throws Exception {
+        assertThat(grpcService.pathMappings())
+            .isEqualTo(
```

Comment:
This test would be clearer if you use containsExactly with the actual path strings, right now it just duplicates the business logic but it's hard to know that logic is actually correct.